crates/telio-model/src/api_config.rs

```diff
@@ -179,16 +191,21 @@ pub struct FeatureDerp {
 }
 
 /// Whether to validate keys
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize)]
-pub struct FeatureValidateKeys(pub bool);
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureValidateKeys {
+    /// Defaults to true
+    pub validate_keys: bool,
+}
```

Comment:
Reminder for myself: this change will require a change in the confluence page for libtelio features

Reply:
I think if you add `repr(transparent)` you will keep the json format unchanged.

Reply:
Yes, I updated these to not have to do custom conversion logic for UniFFI but of course it's better to keep the API simple than to avoid such conversions. Fixed